vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -29,7 +29,7 @@ BAREMETALV6_POOL_RANGE_START: "{{ lookup('env', 'BAREMETALV6_POOL_RANGE_START')
 BAREMETALV6_POOL_RANGE_END: "{{ lookup('env', 'BAREMETALV6_POOL_RANGE_END') | default('fd55::200', true) }}"
 EXTERNAL_SUBNET_V6_PREFIX: "{{ lookup('env', 'EXTERNAL_SUBNET_V6_PREFIX') | default('64', true) }}"
 EXTERNAL_SUBNET_V6_HOST: "{{ lookup('env', 'EXTERNAL_SUBNET_V6_HOST') | default('fd55::1', true) }}"
-IRONIC_HOST_IP: "{{ lookup('env', 'IRONIC_HOST_IP') | default('172.22.0.1', true) }}"
+IRONIC_HOST_IP: "{{ lookup('env', 'IRONIC_HOST_IP') | default('172.22.0.2', true) }}"
```

Comment:
@maelk I think you can remove this change from [#345](https://github.com/metal3-io/metal3-dev-env/pull/345)

Reply:
@fmuyassarov not only this actually, but other files also have same changes as I noticed.

Reply:
That's because #345 was built on top of this PR (check the list of commits)